src/PublicApiGenerator/CodeNormalizer.cs

```diff
@@ -33,6 +34,7 @@ public static string NormalizeGeneratedCode(StringWriter writer)
             gennedClass = Regex.Replace(gennedClass, Set, " { set; }", RegexOptions.IgnorePatternWhitespace);
             gennedClass = Regex.Replace(gennedClass, @"\s+{\s+}", " { }", RegexOptions.IgnorePatternWhitespace);
             gennedClass = Regex.Replace(gennedClass, @"\)\s+;", ");", RegexOptions.IgnorePatternWhitespace);
+            gennedClass = Regex.Replace(gennedClass, $@"(?s)(?<=\[[^\]]*)(Attribute)?({AttributeMarker}(\(\))?(?=\])|{AttributeMarker}(?=\(.+\)\])|{AttributeMarker}(?=new.+\}}\)\]))", string.Empty, RegexOptions.IgnorePatternWhitespace);
```

Comment:
Need multi line here for params object attributes. They cross multiple lines. If you remove it all those attribute test cases fail

Reply:
About the white space. There is. new object[] {...} also the params object attribute

Reply:
> Need multi line here for params object attributes. They cross multiple lines. If you remove it all those attribute test cases fail

My question is, why not use RegexOptions instead of putting it in the regex since you're already using `RegexOptions`?

```suggestion
            gennedClass = Regex.Replace(gennedClass, $@"(?<=\[[^\]]*)(Attribute)?({AttributeMarker}(\(\))?(?=\])|{AttributeMarker}(?=\(.+\)\])|{AttributeMarker}(?=new.+\}}\)\]))", string.Empty, RegexOptions.SingleLine | RegexOptions.IgnorePatternWhitespace);
```

Reply:
> About the white space. There is. new object[] {...} also the params object attribute

`RegexOptions.IgnorePatternWhitespace` means that whitespace in the pattern string is ignored. There is no whitespace in the string `$@"(?s)(?<=\[[^\]]*)(Attribute)?({AttributeMarker}(\(\))?(?=\])|{AttributeMarker}(?=\(.+\)\])|{AttributeMarker}(?=new.+\}}\)\]))"`.

You'd specify IgnorePatternWhitespace if you wanted to indent the regex pattern and leave comments explaining it, inside the pattern string.

Reply:
I should probably add an explanation for our future selfs